aggregator/src/test/resources/application.properties

```diff
@@ -35,6 +49,8 @@ quarkus.log.sentry.dsn=FILL_ME
 
 %test.quarkus.scheduler.enabled=false
 
+email.subscription.daily.cron=1 * * * * ?
```

Comment:
1. I would standardize and use the same prefix: `notifications.aggregator.`
2. Use `periodic` rather than `daily` in the case we add other frequency (I know more work would have to be done in the backend)
3. And use the real cron for daily (again, on your machine for test, you can always override with an environment variable and we should make sure the checked-in code is always correct)

So something like `notifications.aggregator.periodic.cron=0 0 0 * * ?` I guess

Reply:
fixed.